substrate/frame/message-queue/src/mock.rs

```diff
@@ -164,6 +164,7 @@ impl ProcessMessage for RecordingMessageProcessor {
 		meter: &mut WeightMeter,
 		_id: &mut [u8; 32],
 	) -> Result<bool, ProcessMessageError> {
+		sp_io::storage::set(b"transactional_storage", &vec![1, 2, 3]);
```

Comment:
The mocked process supports callbacks - no need to hard code it here.

Check out the tests that use it:  
```rust
Callback::set(Box::new(|_, _| {
	MessageQueue::enqueue_message(mock_helpers::msg("anothermessage"), There);
}));
```